onnxruntime/test/platform/ios/ios_package_test/README.md

```diff
@@ -14,15 +14,15 @@ The iOS End-to-End Test App will use CocoaPods to install the Onnx Runtime C/C++
 ### Model used
 - [sigmoid ONNX model](https://github.com/onnx/onnx/blob/f9b0cc99344869c246b8f4011b8586a39841284c/onnx/backend/test/data/node/test_sigmoid/model.onnx) converted to ORT format
 
-    Here's the [document](http://www.onnxruntime.ai/docs/how-to/deploy-on-mobile.html#1-create-ort-format-model-and-configuration-file-with-required-operators) about how you can convert an ONNX model into ORT format.
+    Here's the [document](onnxruntime.ai/docs/tutorials/mobile/) about how you can convert an ONNX model into ORT format.
```

Comment:
maybe this one instead?
https://onnxruntime.ai/docs/tutorials/mobile/model-conversion.html

Reply:
updated! :)